Assets/Scripts/MarchingCubes/Planet.cs

```diff
@@ -9,25 +9,35 @@ public class Planet : MonoBehaviour
 {
     [SerializeField] private ComputeShader meshGenerator;
     [SerializeField, Range(0, 255)] private float threshold = 200;
-    [SerializeField, Range(1, 28)] private int resolution = 20;
     [SerializeField, Range(1, 25)] private int frequency = 20;
     [SerializeField, Range(0, 5)] private float amplitude = 1;
     [SerializeField, Range(0, 1)] private float bottomLevel = 1;
     [SerializeField] private Material waterMaterial;
     [SerializeField] private GameObject water;
-    [SerializeField] private GameObject meshObj;
 
+    public float diameter;
     public float radius;
     public float surfaceGravity;
     public string bodyName = "TBT";
     public float mass;
     public List<Planet> moons;
 
+    private List<Chunk> chunks;
+    private PillPlayerController player;
```

Comment:
Unused and potential future dependency on PillPlayerController

Reply:
Will be used in Chunks later on to do "LOD's"

Reply:
fixed